ambry-store/src/main/java/com.github.ambry.store/CompactionManager.java

```diff
@@ -98,16 +103,7 @@ void awaitTermination() {
    * @throws StoreException when {@link BlobStore} is not started
    */
   CompactionDetails getCompactionDetails(BlobStore blobStore) throws StoreException {
```

Comment:
can we make this a `private` function now? It will be tested along with the `CompactionExecutor` anyway. The actual policy tests are in the respective tests.